backend/open_webui/routers/images.py

```
@@ -727,3 +766,292 @@ async def image_generations(
+        # Load image(s) from URL(s) if necessary
+        if isinstance(form_data.image, str):
+            form_data.image = await load_url_image(form_data.image)
+        elif isinstance(form_data.image, list):
+            form_data.image = [await load_url_image(img) for img in form_data.image]
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=ERROR_MESSAGES.DEFAULT(e))
+
+    def get_image_file_item(base64_string, param_name="image"):
+        data = base64_string
+        header, encoded = data.split(",", 1)
+        mime_type = header.split(";")[0].lstrip("data:")
```

Ruff (0.14.7)

Using `.strip()` with multi-character strings is misleading

(B005)